storage/pruning/pruningStorer_test.go

```diff
@@ -986,7 +986,7 @@ func TestPruningStorer_processPersistersToClose(t *testing.T) {
 }
 
 func TestPruningStorer_ConcurrentOperations(t *testing.T) {
-	t.Skip("this test should be run only when troubleshooting pruning storer concurrent operations")
+	numOperations := 100 // increase this to 5000 when troubleshooting pruning storer concurrent operations
```

Comment:
since we'll run this test by default, please check that there is no panic on a recover

Reply:
done